src/main/java/teammates/common/util/StatusMessageColors.java

```diff
@@ -0,0 +1,219 @@
+package teammates.common.util;
+
+import java.util.HashMap;
+
+import teammates.common.util.Const.StatusMessageColor;
+import teammates.common.util.Const.StatusMessages;
+
+public class StatusMessageColors {
+    
+    public static HashMap<String, Const.StatusMessageColor> getStatusMessageColorsList() {
+        HashMap<String, Const.StatusMessageColor> statusMessageColors = 
+                                        new HashMap<String, Const.StatusMessageColor>();
+        setStatusMessageColors(statusMessageColors);
+        
+        return statusMessageColors;
+    }
+    
+    public static String getStatusMessageColor(String statusMessage) {
+        try {
+            StatusMessageColor color = getStatusMessageColorsList().get(StringHelper.removeVariables(statusMessage));
+
+            switch (color) {
+                case SUCCESS:
+                    return "success";
+                case WARNING:
+                    return "warning";
+                case DANGER:
+                    return "danger";
+                case INFO:
+                default:
+                    return "info";
+            }
+        } catch (Exception e) {
+            return "info";
+        }
+    }
+
```

Comment:
Consider creating a StatusMessage class that encapsulate color, text etc. If not in this PR, as a separate PR.